src/components/FileInfo/FileInfoComponent.tsx

```diff
@@ -1,15 +1,18 @@
 import * as React from "react";
-import {observer} from "mobx-react";
-import {ControlGroup, Divider, FormGroup, HTMLSelect, IOptionProps, NonIdealState, Pre, Spinner, Tab, TabId, Tabs, Text} from "@blueprintjs/core";
-import {FixedSizeList as List} from "react-window";
+import { observer } from "mobx-react";
+import { Pre, Tab, TabId, Tabs, NonIdealState, Spinner, Text, Label, FormGroup, IOptionProps, HTMLSelect, ControlGroup, Divider, Switch, NumericInput } from "@blueprintjs/core";
+import { FixedSizeList as List } from "react-window";
 import AutoSizer from "react-virtualized-auto-sizer";
-import {CARTA} from "carta-protobuf";
-import {TableComponent, TableComponentProps} from "components/Shared";
+import { CARTA } from "carta-protobuf";
+import { TableComponent, TableComponentProps } from "components/Shared";
 import "./FileInfoComponent.scss";
+import { AppStore, FileBrowserStore } from "stores";
+import { SpectralSystem, SpectralType, SpectralUnit } from "models";
 
 export enum FileInfoType {
     IMAGE_FILE = "image-file",
     IMAGE_HEADER = "image-header",
+    CHOP_IMAGE = "chop-image",
```

Comment:
I don't think this really belongs in the file info component, since it's more about settings for saving a file. I suggest we move this into a separate component. I also think we should use `SAVE_SUBIMAGE` instead of `CHOP_IMAGE`

Reply:
Changed the naming

Reply:
How about add a new component for image save, 'ImageSaveComponent' or 'FileSaveComponent'?

Reply:
May I have any hint about how to refactor it or any suggestion? I suppose to move out the codes related to 'save image' to a new component, 'components/SaveFile/ImageSaveComponent'.

Reply:
I'd say `FileBrowser/ImageSaveComponent` would work

Reply:
Got it.